tests/common.go

```diff
@@ -897,6 +897,12 @@ func ValidateContext(ctx *scheduler.Context, errChan ...*chan error) {
 			if strings.Contains(ctx.App.Key, "tektoncd") {
 				ctx.SkipVolumeValidation = true
 			}
+			if strings.Contains(ctx.App.Key, "pxb-singleapp-multivol") {
+				ctx.SkipVolumeValidation = true
+			}
+			if strings.Contains(ctx.App.Key, "pxb-multipleapp-multivol") {
+				ctx.SkipVolumeValidation = true
+			}
```

Comment:
Create a single slice with all the apps which needs volume validation to be skipped

Reply:
Done